GUI/MainWindow.xaml.cs

```diff
@@ -1409,5 +1412,18 @@ private void AddContaminantXML_Click(object sender, RoutedEventArgs e)
             }
             dataGridProteinDatabases.Items.Refresh();
         }
+
+        // handle window closing
+        private void MainWindow_Closing(object sender, CancelEventArgs e)
```

Comment:
would be nice to have a "never ask me again" button. It sounds like that means making a custom window but that's ok if it's needed

Reply:
yeah i tried to add that but i'd have to create a custom message button. should i still do it?

Reply:
it would involve creating a new CustomMessageBox class - so idk if that's too much code

Reply:
yeah just make that custom class. might be nice to have around for other purposes too